js/src/Builder.js

```diff
@@ -0,0 +1,1754 @@
+/**
```

Comment:
You're going to have to merge in the latest version of Escher to get a realistic diff here. Let me know if I can help with that.

Reply:
For one thing, source files are now in `src/` instead of `js/src/`

Reply:
Wait, ignore those comments: I think you just need to delete this old version of `Builder.js`.